src/components/BannerImage/BannerImage.styles.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+import styled from "styled-components";
+
+export const BannerContainer = styled.header`
+  width: 100%;
+  position: relative;
+  overflow: hidden;
+
+  @media (max-width: 768px) {
+    height: 300px;
+  }
+
+  @media (max-width: 480px) {
+    height: 200px;
+  }
+`;
+
+export const Banner = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
```

Comment:
A max height would suit this well

```suggestion
  max-height: 20rem;
  object-fit: cover;
```